lte/gateway/c/core/oai/test/amf/test_amf_stateless.cpp

```diff
@@ -0,0 +1,299 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <chrono>
+#include <gtest/gtest.h>
+#include <thread>
+
+#include "../mock_tasks/mock_tasks.h"
+
+extern "C" {
+#include "lte/gateway/c/core/oai/common/common_types.h"
+#include "lte/gateway/c/core/oai/include/amf_config.h"
+}
+#include "lte/gateway/c/core/oai/tasks/amf/include/amf_client_servicer.h"
+#include "lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context_and_proc.h"
+#include "lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.h"
+#include "lte/gateway/c/core/oai/test/amf/amf_app_test_util.h"
+
+using ::testing::Test;
+
+namespace magma5g {
+
+extern task_zmq_ctx_s amf_app_task_zmq_ctx;
+
+TEST(test_state_converter, test_guti_to_string) {
```

Comment:
Please use camel case for naming the test suite

```suggestion
TEST(TestAMFStateConverter, TestGutiToString) {
```

Reply:
Yes sure, will maintain this practice going forward. Review comment addressed